op-node/rollup/driver/origin_selector.go

```diff
@@ -68,12 +75,21 @@ func (los *L1OriginSelector) FindL1Origin(ctx context.Context, l2Head eth.L2Bloc
 		return currentOrigin, nil
 	}
 
+	receiptsCached := true
+	receiptsCtx, receiptsCancel := context.WithTimeout(ctx, 10*time.Millisecond)
```

Comment:
10 milliseconds is a bit short, because the FetchReceipts method also requests eth_getBlockByHash, which can affect response time if it is not cached. In addition, relying on response time to determine if it is cached is unreliable, as sometimes even if it is cached, the response time may exceed a certain value due to eth_getBlockByHash or other reasons (gc?). It is recommended to modify the return value and add a variable to inform the caller if the cache is hit, which would be more reasonable.

Reply:
Ok, so here should be a naming issue. Since we don't care about whether it is cached or not, let's change the variable name, for example, call it: isReceiptsFast. I think this is more intuitive, of course it's not a core issue, just a suggestion.